### DISCUSSION TITLE
Support of Arabic language in tts

### DISCUSSION THREAD

**Discussion by nik0811** [deepgram/community#982] [General help] [answered]
Are Deepgram planing to support Arabic language in text to speech, as we have a requirements from the customer, currently because of not having this support we need to reinvent a wheel to switch TTS on elevenlabs.

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- A request ID that triggered your error or issue.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @nik0811, currently Deepgram only supports English text-to-speech, and we do not have Arabic TTS on our roadmap at this time.
